shippable/install.sh

```diff
@@ -34,7 +34,7 @@ python-dev=2.7*
 
 pip install -q virtualenv==16.5.0
 pip install -q pyOpenSSL==19.0.0
-
+pip install statistics numpy enum34 #Needed by EDTT
```

Comment:
`pip install -q` just looking at what we do a few lines above.

Reply:
more silent it is :)